src/components/Markdown.svelte

```diff
@@ -79,6 +101,15 @@
     }
 
     await Promise.allSettled(treeChanges);
+
+    return () => {
```

Comment:
I think we don't need to clean up anything here, because it wouldn't work on an `async` function anyway:

> This behaviour will only work when the function passed to onMount synchronously returns a value. async functions always return a Promise, and as such cannot synchronously return a function.

and there wouldn't be anything to click on when the component is destroyed, so I think any click handlers would be cleaned up automatically.

Reply:
Yeah you're right!